docs/man/docker-create.1.md

```diff
@@ -67,6 +68,12 @@ IMAGE [COMMAND] [ARG...]
 **--cpuset**=""
    CPUs in which to allow execution (0-3, 0,1)
 
+**--dev**=host
+   Use the system's /dev inside of the container
+     **host**: use the host's /dev directory inside the container.
+     No devices will be mounted on the /dev or any of its subdirs, from inside of the container
+     Note: the host mode gives the container full access to host devices and is therefore considered insecure.
```

Comment:
how full, and how insecure? is the container able to create and modify devices? `No devices will be mounted ...` line still leaves me with the feeling that `/dev` is mounted into the container in some magical read-only way, but `full access` feels like it contradicts that.